app/views/content/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -76,63 +83,72 @@ If you do not have a degree yet, you can apply to an [undergraduate teacher trai
 
 #### English and maths qualifications
 
-You usually need the equivalent of a GCSE (grade 4) in English and maths.
+You will need the equivalent of a GCSE (grade 4) in English and maths.
 
 To teach children aged 3 to 11, you also need the equivalent of a GCSE (grade 4) in a science subject.
 
-#### Help with international qualifications
- 
+#### We can help if you have international qualifications
+
 If your qualifications come from a non-UK institution, your teacher training provider may want to see a ‘[statement of comparability](https://enic.org.uk/Qualifications/SOC/Default.aspx)’ showing their equivalence to UK qualifications.
 
 Call us on [0800 389 2500](tel://08003892500) for:
 
 * guidance on the UK equivalents of your qualifications
-* a free statement of comparability from UK ENIC once you’ve submitted your application, if your provider asks for this
+* a free statement of comparability detailing overseas degree equivalency from UK ENIC once you’ve submitted your application, if your provider asks for this
 
-#### An offer to train
+### 2. Get a teacher training adviser
 
-Explore different [ways to train](/ways-to-train) and consider whether you want do a training programme which leads to [qualified teacher status (QTS)](/ways-to-train). QTS allows you to apply to all teaching jobs in England.
+You can get an adviser for one-to-one support. They can give you practical advice and guidance at every stage of your application.
 
-Fees may be higher for international trainees but you may be able to [get financial support](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-non-uk-applicants#financial-support-for-non-uk-applicants-for-unsalaried-teacher-training-in-england).
+<p class="call-to-action__action">
```

Comment:
@peteryates Can you check whether buttons on this page should be aligned with the text?